azext_iot/tests/iothub/devices/test_hub_export_import_int.py

```diff
@@ -16,19 +17,23 @@
 from azext_iot.tests.iothub import (
     PRIMARY_THUMBPRINT,
     SECONDARY_THUMBPRINT,
-    DATAPLANE_AUTH_TYPES,
     DEVICE_TYPES,
 )
 
 settings = DynamoSettings(req_env_set=ENV_SET_TEST_IOTHUB_REQUIRED, opt_env_set=ENV_SET_TEST_IOTHUB_OPTIONAL)
 CWD = os.path.dirname(os.path.abspath(__file__))
 
+# changing the order so the final exported file can be used for all import tests
+# (the cstring output doesn't include the keys for the control plane features)
+DATAPLANE_AUTH_TYPES = ["cstring", "key", "login"]
```

Comment:
nit, why not use AuthenticationTypeDataplane enum attributes in common.shared rather than hard coded strings?